examples/python_yolov4/yolov4_inference.ipynb

```diff
@@ -0,0 +1,201 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Object Detection with YoloV4\n",
+    "This notebook is intended to be an example of how to use MIGraphX to perform object detection. The model used below is a pre-trained yolov4 from the ONNX model zoo. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Download dependencies"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os.path\n",
+    "from os import path\n",
+    "\n",
+    "if not path.exists(\"coco.names\"):\n",
+    "    !wget https://github.com/onnx/models/raw/master/vision/object_detection_segmentation/yolov4/dependencies/coco.names\n",
+    "if not path.exists(\"yolov4_anchors.txt\"):\n",
+    "    !wget https://github.com/onnx/models/raw/master/vision/object_detection_segmentation/yolov4/dependencies/yolov4_anchors.txt\n",
+    "if not path.exists(\"input.jpg\"):\n",
+    "    # The image used is from the COCO dataset (https://cocodataset.org/#explore)\n",
+    "    # Other images can be tested by replacing the link below\n",
+    "    image_link = \"https://farm3.staticflickr.com/2009/2306189268_88cc86b30f_z.jpg\"\n",
+    "    !wget -O input.jpg $image_link\n",
+    "if not path.exists(\"yolov4.onnx\"):\n",
+    "    !wget https://github.com/onnx/models/raw/master/vision/object_detection_segmentation/yolov4/model/yolov4.onnx"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Serialize model using model optimizer tool\n",
+    "Please refer to the [model optimizer example](../python_model_optimizer/README.md) if you would like more information about this tool."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "if not path.exists(\"yolov4_fp16.msgpack\"):\n",
+    "    !python3 ../python_model_optimizer/optimize_model.py -t gpu -f -p ../python_yolov4/yolov4_fp16 ../python_yolov4/yolov4.onnx\n",
+    "if not path.exists(\"yolov4.msgpack\"):\n",
+    "    !python3 ../python_model_optimizer/optimize_model.py -t gpu -p ../python_yolov4/ ../python_yolov4/yolov4.onnx"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Import libraries \n",
+    "Please refer to [this section](https://github.com/ROCmSoftwarePlatform/AMDMIGraphX#using-migraphx-python-module) of the main README if the migraphx module is not found. "
```

Comment:
I find it better to just add logic to add path here. it is 3-4 lines.

Reply:
They may also need to create the deb package, so I would prefer to refer to the main readme.